Nos/Views/Moderation/ContentFlagView.swift

```diff
@@ -92,26 +92,26 @@ struct ContentFlagView: View {
             }
         }
     }
+}
 
-    private var successView: some View {
-        VStack(spacing: 30) {
-            Image.circularCheckmark
-                .resizable()
-                .aspectRatio(contentMode: .fit)
-                .frame(height: 116)
+var flagSuccessView: some View {
```

Comment:
I'm surprised this isn't inside of the ContentFlagView type. Could we put it in there?

Reply:
Oh, I see that it's used by UserFlagView, too. Could it go into its own View type?

Reply:
I thought about this @joshuatbrown, but it isn't used in the entire app. What do you think about a components folder within the moderation folder?

Reply:
I'd just put it in the Moderation folder and not worry about an additional components folder inside it for now -- if I'm seeing the right place, we only have one file in Moderation right now.

Reply:
Alright that makes sense @joshuatbrown. What about a case where multiple views share the exact same function? Do we duplicate those functions in the different views?

Reply:
We discussed this on a call, but I'll post the summary here, too. I wouldn't duplicate. For this, I'd probably make a FlagSuccessView that's in the Moderation folder that gets used by both ContentFlagView and UserFlagView. For functions, I'd also put them in a place that can be shared by multiple views (if multiple views need the function).